modules/http4s/src/smithy4s/http4s/SimpleRestJsonBuilder.scala

```diff
@@ -17,7 +17,11 @@
 package smithy4s
 package http4s
 
+import smithy4s.internals.InputOutput
+
 object SimpleRestJsonBuilder
     extends SimpleProtocolBuilder[smithy4s.api.SimpleRestJson](
-      smithy4s.http.json.codecs
+      smithy4s.http.json.codecs(
+        smithy4s.api.SimpleRestJson.protocol.hintMask ++ HintMask(InputOutput)
```

Comment:
**nitpick (if-minor):** Kind of makes me think the standard hints (smithy.api) should always be forwarded no matter what.

Reply:
Nah : imagine you want to have a json RPC API and also serve it over http/restJson. You need a mechanism to ignore the http binding traits when using the RPC protocol (or re-implement a bespoke schematic)

Reply:
seems like `smithy.api` is too generic a package then 😅